src/session/Session.ts

```diff
@@ -497,3 +498,18 @@ function setupCompletionHandler(
   }
   return { completionHandler, targetElement, properties };
 }
+
+function getConnectionData(token: string): string {
+  if (!token) return "";
```

Comment:
Returning an empty string instead of undefined is the right way to go from what I've seen in other Opentok demos